docs/src/content/overview-features.md

```diff
@@ -41,30 +41,22 @@ conversation, where requests may have been made concurrently.
 You may want to use client-side replay in conjunction with the `anticache`
 option, to make sure the server responds with complete data.
 
-## Proxy Authentication
-
-Asks the user for authentication before they are permitted to use the proxy.
-Authentication headers are stripped from the flows, so they are not passed to
-upstream servers. For now, only HTTP Basic authentication is supported. The
-proxy auth options are not compatible with the transparent, socks or reverse
-proxy mode.
-
 
-## Replacements
+## Modify Body
 
-The `replacements` option lets you specify an arbitrary number of patterns that
-define text replacements within flows. A replacement pattern looks like this:
+The `modify-body` option lets you specify an arbitrary number of patterns that
```

Comment:
`modify-body` is correct. `argparse` replaces `_` with `-`.

Reply:
We auto-translate `_` to `-` for command line arguments, yes. However, `modify_body` is what you would set in your config.yaml or in mitmproxy's options editor, and what we consistently use in our docs (see https://docs.mitmproxy.org/stable/concepts-options/). You could also do `--set modify_body=...` for that matter. The inconsistency is a bit ugly, but there really aren't many better options around.

Reply:
(if you have any ideas on how we could have this more consistent, I'd love to hear them - what we currently have is the best solution we came up with so far)

Reply:
Thanks for clarification.

Reply:
It might make sense to inform the user when underscore-formatted options are used as a cli argument, similar to the messages on deprecated commands.
At the moment it outputs: `mitmproxy: error: unrecognized arguments: --modify_body asdf`.

Reply:
Agreed, that's actually a really good point! Let's track this in #4054.